src/styles/index.scss

```diff
@@ -118,104 +14,15 @@ body {
 	justify-content: center;
 	align-items: center;
 	border: 3px solid #2ecc71;
-	&:hover {
-		z-index: 2;
-		color: #cdd6e0;
-		text-decoration: none;
-		background-color: #343a40;
-		border: 3px solid #2ecc71;
-	}
-}
-
-.card-box {
-	background-color: #222121;
-	// border: 1px solid rgb(66, 62, 62);
-	padding: 20px;
-	margin-bottom: 20px;
-	width: 100%;
-	border-radius: 5px;
-	box-shadow: 5px 5px 10px rgba(27, 27, 27, 0.5);
 }
-
-.card-box > a {
-	color: inherit;
-	&:hover {
-		text-decoration: none;
-	}
-}
-
-.company-detail {
-	// border: 1px solid white;
-	width: 100%;
-	text-align: center;
-	border-bottom: 1px solid #696565;
-	margin-bottom: 3px;
-}
-
-.company-detail img {
-	height: 98px;
-	width: 98px;
-}
-
-.job-description {
-	display: flex;
-	align-items: center;
-	flex-direction: column;
-
-ul {
-	display: flex;
-	padding: 0;
-}
-
-li {
-	display: flex;
-	margin: 6px;
-}
-
-}
-
-.badge-time svg,
-.badge-location svg {
-	font-size: 20px;
-}
-
-.view-btn {
-	display: inline-block;
-	background: #044902;
-	padding: 0.45em 1em;
-	width: 100%;
-	text-align: center;
-	border-radius: 5px;
-	color: #fff;
-	font-weight: 500;
-	letter-spacing: 0.2px;
-	font-size: 17px;
-	&:hover {
-		background: #097205;
-	}
-}
-// Small devices (landscape phones, 576px and up)
-@media (min-width: 576px) {
-}
-
-// Medium devices (tablets, 768px and up)
-@media (min-width: 768px) {
-	.search-form {
-		display: flex;
-		flex-direction: column;
-	}
-	.main-body h2 {
-		text-align: center;
-		color: #58a6ff;
-		margin: 8px;
-		display: block;
-	}
+.page-link:hover {
+	z-index: 2;
+	color: #cdd6e0;
+	text-decoration: none;
+	background-color: #343a40;
+	border: 3px solid #2ecc71;
 }
 
-// Large devices (desktops, 992px and up)
-@media (min-width: 992px) {
-	.search-form {
-		display: flex;
-		flex-direction: row;
-	}
+.mobile-search .btn{
+	border-radius: 50%;
```

Comment:
Should remove all SCSS

Reply:
`.page-link,
.page-item:first-child .page-link,
.page-item:last-child .page-link {
	width: 40px;
	display: flex;
	height: 40px;
	color: #bf8c4f;
	background-color: #262727;
	line-height: 40px;
	border-radius: 100%;
	padding: 0;
	position: relative;
	justify-content: center;
	align-items: center;
	border: 3px solid #2ecc71;
}
.page-link:hover {
	z-index: 2;
	color: #cdd6e0;
	text-decoration: none;
	background-color: #343a40;
	border: 3px solid #2ecc71;
}`

i left-out this part because i have a question that i had asked on the styled bootstrap elements ....do you remembeer asking me why i cant only use `page-link` ??